TAO/docs/compiler.html

```diff
@@ -187,316 +255,327 @@ <h3>Operation Demuxing Strategies</h3>
 the following:
 
 <ul>
-  <LI>Enable <CODE>ACE_HAS_GPERF</CODE> when building ACE and TAO.
-      This macro has been defined for the platforms where <CODE>gperf</CODE>
+  <li>Enable <code>ACE_HAS_GPERF</code> when building ACE and TAO.
+      This macro has been defined for the platforms where <code>gperf</code>
       has been tested, which includes most <A
       HREF="http://www.cs.wustl.edu/~schmidt/ACE-versions-i.html">platforms</A>
-      that ACE runs on. <P>
+      that ACE runs on.</li>
 
-  <li>Build the <CODE>gperf</CODE> in $ACE_ROOT/apps/gperf.  This build
+  <li>Build the <code>gperf</code> in $ACE_ROOT/apps/gperf.  This build
       also leaves a copy/link of the ace_gperf program at the $ACE_ROOT/bin
-      directory.<P>
+      directory.</li>
 
-  <li>Set the environment variable $ACE_ROOT appropriately or add
-      $ACE_ROOT/bin to your search path. <P>
+  <li>Set the environment variable <code>$ACE_ROOT</code> appropriately or add
+      <code>$ACE_ROOT/bin</code> to your search path.</li>
 
-  <li>Use the <CODE>-g</CODE> option for the TAO IDL compiler or set
-      your search path accordingly to install <CODE>ace_gperf</CODE> in a
-      directory other than $ACE_ROOT/bin.
+  <li>Use the <code>&#8209;g</code> option for the TAO IDL compiler or set
+      your search path accordingly to install <code>ace_gperf</code> in a
+      directory other than <code>$ACE_ROOT/bin</code>.
 </ul>
 
 Note that if you can't use perfect hashing for some reason the next
 best operation demuxing strategy is binary search, which can be
 configured using TAO's IDL compiler <A HREF="#options">options</A>.<P>
 
-<HR><P>
-<h3>AMI support</h3>
+<hr>
+<h2>AMI support</h2>
 
 The TAO_IDL compiler generates AMI stubs and skeletons as
 described in the CORBA 3.0.3 specification.
 
-<hr><p>
-<h3><a name="collocation-stubs">Collocation Strategies</a></h3>
+<hr>
+<h2 id="collocation-stubs">Collocation Strategies</h2>
 
+<p>
 TAO_IDL can generate collocated stubs using two different collocation
 strategies.  It also allows you to suppress/enable the generation of
 the stubs of a particular strategy.  To gain great flexibility at
 run-time, you can generate stubs for both collocation strategies
-(using both '<a href="#Gp"><code>-Gp</code></a>' and '<a
-href="#Gd"><code>-Gd</code></a>' flags at the same time) and defer the
+(using both '<a href="#Gp"><code>&#8209;Gp</code></a>' and '<a
+href="#Gd"><code>&#8209;Gd</code></a>' flags at the same time) and defer the
```

Comment:
Why is this changing?

Reply:
These are non-breaking dashes, which are like [non-breaking spaces](https://en.wikipedia.org/wiki/Non-breaking_space). They prevent line breaks around the dash which was making the page look weird in some sections, especially in the options table, where it would look something like this:

```
--
idl-version VERSION
```